src/crepe/api/Camera.h

```diff
@@ -20,33 +21,34 @@ class Camera : public Component {
 	 * \param id Unique identifier for the camera component.
 	 * \param bg_color Background color for the camera view.
 	 */
-	Camera(game_object_id_t id, const Color & bg_color);
+	Camera(game_object_id_t id, const Color & bg_color, const ivec2 & screen, const ivec2 & viewport, const double & zoom);
```

Comment:
Mist `offset` niet nog in de constructor? Eventueel voeg je die toe aan de constructor, maar dan met default waardes in de constructor (zodat het nog steeds mogelijk is om offset niet door te geven).